influxdb/client.py

```diff
@@ -495,6 +495,32 @@ def drop_database(self, dbname):
         """
         self.query("DROP DATABASE {0}".format(quote_ident(dbname)))
 
+    def get_list_measurements(self):
+        """Get the list of measurements in InfluxDB.
+
+        :returns: all measurements in InfluxDB
+        :rtype: list of dictionaries
+
+        :Example:
+
+        ::
+
+            >> dbs = client.get_list_measurements()
+            >> dbs
+            [{u'name': u'measurements1'},
+             {u'name': u'measurements2'},
+             {u'name': u'measurements3'}]
+        """
+        return list(self.query("SHOW MEASUREMENTS").get_points())
```

Comment:
Would be good to add support for `on` clause which was added in InfluxDB 1.1.

Reply:
I concur with that, there are features added regularly that we'll need to keep up with. Since we're officially supporting the v1.2.4 release this is fine as-is.